topics/single-cell/faqs/sc_version.md

```diff
@@ -0,0 +1,63 @@
+---
+redirect_from:
+- /topics/transcriptomics/faqs/amplification
```

Comment:
is this intended or a copy/paste from the amplification FAQ?

(Also, it is better to create symlinks if you want an FAQ to show up in 2 places, this should just be used if you have moved or renamed the physical file from this location)

Reply:
Oh good to know about symlinks! Does that work for tutorials as well?

Reply:
yep, works for any FAQs :+1: